pkg/gofr/errors/gofrError.go

```diff
@@ -0,0 +1,37 @@
+package errors
```

Comment:
should we have package name as gofrError, as this might collide with actual package name.

Just a thought!

Reply:
sure will rename it